packages/espresso/example/android/app/src/androidTest/java/com/example/MainActivityTest.java

```diff
@@ -1,76 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-package com.example.espresso_example;
-
-import static androidx.test.espresso.flutter.EspressoFlutter.onFlutterWidget;
-import static androidx.test.espresso.flutter.action.FlutterActions.click;
-import static androidx.test.espresso.flutter.action.FlutterActions.syntheticClick;
-import static androidx.test.espresso.flutter.assertion.FlutterAssertions.matches;
-import static androidx.test.espresso.flutter.matcher.FlutterMatchers.withText;
-import static androidx.test.espresso.flutter.matcher.FlutterMatchers.withTooltip;
-import static androidx.test.espresso.flutter.matcher.FlutterMatchers.withValueKey;
-import static com.google.common.truth.Truth.assertThat;
-
-import androidx.test.core.app.ActivityScenario;
-import androidx.test.espresso.flutter.EspressoFlutter.WidgetInteraction;
-import androidx.test.espresso.flutter.assertion.FlutterAssertions;
-import androidx.test.espresso.flutter.matcher.FlutterMatchers;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-/** Unit tests for {@link EspressoFlutter}. */
-@RunWith(AndroidJUnit4.class)
-public class MainActivityTest {
-
-  @Before
-  public void setUp() throws Exception {
-    ActivityScenario.launch(MainActivity.class);
-  }
-
-  @Test
-  public void performTripleClick() {
-    WidgetInteraction interaction =
-        onFlutterWidget(withTooltip("Increment")).perform(click(), click()).perform(click());
```

Comment:
Could you file an issue referencing this deleted code so we don't lose track of the existence of this native integration test? We should figure it why they are broken and fix them (just not in this PR).